CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 48 Release Notes
 
+### Bug fixes
+
++ Fix a bug where zip files with directories could not be imported. 
+  For example a zip with `a.wdl` and `b.wdl` could be imported but one with `sub_workflows/a.wdl` 
+  and `imports/b.wdl` could not.
```

Comment:
It would be nice to have a test that demonstrates this is fixed, though I'm not sure how much work that would be.

Reply:
Well given that it is a function that I want to make the distribution of BioWDL pipelines dependent on, the amount of work is not a stopper. These zips are so much nicer than having to check out git repositories, making sure all the submodules are correctly up to date etc. Also these git repositories contain a lot of cruft from the user perspective: test files, CI stuff etc. 
I have added a test with a simple zip file created from the BioWDL QC pipeline. It turned out to be quite easy to test.